pytket/docs/changelog.rst

```diff
@@ -10,10 +10,13 @@ API changes:
   ``map`` property instead.)
 * The deprecated ``Backend.compile_circuit`` method is removed. (Use
   ``get_compiled_circuit`` instead.)
+* ``RebaseCustom`` takes one allowed gateset parameter rather than separate single qubit and multiqubit gatesets.
 
 Minor new features:
 
 * Add ``delay_measures`` option to ``DefaultMappingPass``.
+* New ``pytket.passes.auto_rebase_pass`` which attempts to construct a rebase pass given a target gate set from known decompositions.
+* New ``pytket.circuit.pool`` module containing functions to generate common circuits.
```

Comment:
```suggestion
* New ``pytket.circuit.library`` module containing functions to generate common circuits.
```

Reply:
removing the entire line because the module is private